content/gettingstarted/index.adoc

```diff
@@ -22,123 +22,125 @@ Kiali will support the latest Istio version (1.1 at the moment) and the developm
 is closely following the development of Istio.
 Older Istio versions like 1.0.x may continue to work, but will no longer be actively supported.
 
-== Getting started on OpenShift
+== Installing Kiali
 
-To deploy Kiali to your OpenShift cluster you can do the following.
+To deploy Kiali, you may need to prepare your cluster environment.
 
-=== Preparation
+=== OpenShift Preparation
 
-First, you need to grant the user that is installing Istio and Kiali the `cluster-admin` role.
-
-If OpenShift is installed locally on the machine you are using, the following command should log you in as a user with the `cluster-admin` role:
+If you are installing on OpenShift, you need to grant the `cluster-admin` role to the user that is installing Istio and Kiali.  If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role:
 
 [source,bash]
 ----
 oc login -u system:admin
 ----
 
-You can now install Istio if needed. See the link:https://istio.io/docs/setup/[Istio Setup docs] for details.
-
-=== Install Kiali
-
-Now install the latest release of Kiali.
+=== Kubernetes Preparation
```

Comment:
it looks a bit as if this is a followup/sequential step after openshift preparation and not instead of it. the numbering also supports that. it will be good to emphasize it's only one of them

Reply:
I tweeked this section to hopefully make things more clear